dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
@viveknathani  Can you update the file with execute mode permissions and commit again?

Reply:
Just to be clear, you want me to apply, 
```bash
git update-index --chmod=+x dev.sh
```
and then make the commit?

Reply:
Yea

Reply:
Done